packages/explorer/src/components/playground/Playground.tsx

```diff
@@ -0,0 +1,241 @@
+  const [exampleVars, setExampleVars] = useState<any[]>([])
+  
+  useEffect(() => {
+    initFlowbite()
+  }, [])
+
+  const clearExamples = () => {
+    setExampleCode("")
+    setExampleExpresion("")
+    setExampleModule("")
+    setExampleVars([])
+  }
+  const loadExamples = (type: string) => {
+    switch (type) {
+      case "nft":
+        setExampleCode(nft)
+        setExampleExpresion(nftExpresion)
+        setExampleModule(nftExport)
+        setExampleVars(nftVars)
+        break
+      case "token":
+        setExampleCode(fungibleToken)
+        setExampleExpresion(fungibleTokenExpresion(computer.getPublicKey()))
+        setExampleModule(fungibleTokenExport)
+        setExampleVars(tokenVars(computer.getPublicKey()))
+        break
+      case "chat":
+        setExampleCode(chat)
+        setExampleExpresion(chatExpresion)
+        setExampleModule(chatExport)
+        setExampleVars(chatVars)
+        break
+      case "counter":
+        setExampleCode(counter)
+        setExampleExpresion(counterExpresion)
+        setExampleModule(counterExport)
+        setExampleVars(counterVars)
+        break
+      default:
+        console.log("Please select valid type")
+    }
+  }
+
+  return (
+    <>
+      <div className="pt-4 w-full relative">
+        <Tabs />
+
+        <div id="default-tab-content">
+          <div className="hidden" id="create-new" role="tabpanel" aria-labelledby="create-new-tab">
+            <CreateNew
+              computer={computer}
+              setShow={setShow}
+              setFunctionCallSuccess={setFunctionCallSuccess}
+              setFunctionResult={setFunctionResult}
+              exampleCode={exampleCode}
+              exampleVars={exampleVars}
+            />
+          </div>
+          <div
+            className="hidden"
+            id="execute-expressions"
+            role="tabpanel"
+            aria-labelledby="execute-expressions-tab"
+          >
+            <ExecuteExpression
+              computer={computer}
+              setShow={setShow}
+              setFunctionCallSuccess={setFunctionCallSuccess}
+              setFunctionResult={setFunctionResult}
+              exampleExpression={exampleExpression}
+              exampleVars={exampleVars}
+            />
+          </div>
+          <div
+            className="hidden"
+            id="deploy-modules"
+            role="tabpanel"
+            aria-labelledby="deploy-modules-tab"
+          >
+            <DeployModule
+              computer={computer}
+              setShow={setShow}
+              setFunctionCallSuccess={setFunctionCallSuccess}
+              setFunctionResult={setFunctionResult}
+              exampleModule={exampleModule}
+            />
+          </div>
+        </div>
+
+        <hr className="h-px my-8 bg-gray-200 border-0 dark:bg-gray-700" />
+
+        <Examples loadExamples={loadExamples} clearExamples={clearExamples} />
+
+      </div>
+      <ModalOld
```

Comment:
Here we are using the ModalOld, maybe we should change the name, or use the new Modal component